src/Components/ImagesTable/ImageLink.js

```diff
@@ -62,12 +63,13 @@ const ProvisioningLink = ({ imageId, isExpired, isInClonesTable }) => {
 const ImageLink = ({ imageId, isExpired, isInClonesTable }) => {
   const image = useSelector((state) => selectImageById(state, imageId));
   const uploadStatus = image.uploadStatus;
+  const { initiated: chromeInitiated, isBeta, getEnvironment } = useChrome();
```

Comment:
I think `initiated` is a typo, shouldn't it be `initialized`? 

Doesn't work for me with `initiated` but does work with `initialized`...